app/modals/ViewEvent/Content/index.js

```diff
@@ -93,10 +93,14 @@ const EventContent = memo(({ event, onClose }) => {
           <strong>Organiser:</strong>
         </p>
         <AttendeeItem className="mb-4" invite={organiserInvite} event={event} />
-        <p>
-          <strong>Attendees:</strong>
-        </p>
-        <AttendeesList className="mb-4" event={event} />
+        {event.invites && (
```

Comment:
!![] is true. What is the array is empty?

Reply:
So can i check the array & length instead?
like: ([].isArray([]) && [].length)

Reply:
`!!event.invites?.length`

Reply:
Yeah exactly.. so that'll work right?

Reply:
yes

Reply:
Done!